sgf-viewer/src/components/Section.svelte

```diff
@@ -1,22 +1,32 @@
 <script lang="ts">
-    export let section: Map<string, string>;
+    import { setContext } from "svelte";
+    import { writable } from "svelte/store";
```

Comment:
* svelte context = define variables visible to this component and its descendants
* svelte store = define reactive variables that can be shared between components
* context + store = scoped shared reactive variables